scope3_methodology/cli/model_networking_emissions.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+""" Compute emissions for a networking emissions """
+import argparse
+import logging
+
+from scope3_methodology.api.input_models import NetworkingConnectionType
+from scope3_methodology.networking.model import NetworkingConnection
+from scope3_methodology.utils.yaml_helpers import yaml_dump
+
+
+def parse_args():
+    """Parse the command line arguments"""
+    # arse command line to get company file
+    parser = argparse.ArgumentParser(description="Compute emissions for an publisher")
+    parser.add_argument(
+        "-f",
+        "--defaultsFile",
+        default="defaults/networking-defaults.yaml",
+        help="Set the defaults file to use (overrides networking-defaults.yaml)",
+    )
+    parser.add_argument(
+        "-c",
+        "--connection",
+        const=1,
+        default=NetworkingConnectionType.UNKNOWN.value,
```

Comment:
rather than unknown as a possible value, can it just be that if connection type is not specified, we use the model for an unknown connection. so, null as the value rather than explicitly saying `unknown`

Reply:
We directly reference unknown in the docs, and we have a template for it because we specify specific defaults. So I think there should also be a specified unknown connection type.